types/dom-css/dom-css-tests.ts

```diff
@@ -0,0 +1,33 @@
+import css = require('.');
```

Comment:
sorry, this have to be proper:
```suggestion
import css = require('dom-css');
```

Reply:
All good 👍, I forgot about this again 😅